---
title: "Phylogenetics Seminar"
subtitle: Spring 2024
---

* Biol 4750/6750
* Time: Wednesdays at 10:30
* Class website: [https://crothfels.github.io/PhyloSeminar/years/2024/2024](https://crothfels.github.io/PhyloSeminar/years/2024/2024)


## Instructors

* Carl Rothfels -- carl.rothfels@usu.edu
* Mike May -- mrmay@berkeley.edu, _in absentia_


## Format

I will lead a series of introductory sessions (lectures) for the first three or four meetings. Subsequent meetings will be student-led. Each student will be responsible for leading a session (potentially in groups, depending on enrollment and interest) on a topic of their choice. That student will be responsible for developing an annotated bibliography of 10 – 20 important papers on the topic, including both “classics” and the most recent works, for selecting two of the papers for the class to read, and for preparing a short (15min) presentation on the topic. The remainder of the session will be devoted to discussion.

Possible topics include: modeling migration/hybridization/introgression (phylogenetic networks); state-dependent diversification models; fossil data (e.g. fossilized-birth-death process/tip dating vs node calibrations); modeling gene duplication and loss; polymorphism-aware (PoMo) models; covarion/hidden state models; model testing (e.g. Bayes factors, AIC); model adequacy (e.g. posterior predictive tests); demographic inference (e.g., inferring population size changes through time); the multispecies coalescent, genetrees/species tree; the BAMM controversy; divergence time dating; inferring selection from sequence alignments; alignment inference (alignment-phylogeny co-inference?); community phylogenetics; spatial phylogenetics; inferring phylogenies from morphological data; heterogeneous models, e.g., non-stationary base frequencies; ancestral state reconstruction; phylogenomics (strengths, pitfalls?); inferring correlated evolution of traits; assessing support; hypothesis testing of relationships; reconstructing morphological evolution on a phylogeny; ABC approaches; hidden Markov models; mixture models; species delimitation; microbial community analysis; quartet-based methods (e.g. SVDquartets); etc; etc. Also more practical topics would make cool sessions, such as "how to use YYYY software", etc. Effectively anything phylogenetics-related would be great; please consult with me prior to picking a topic so that we can make sure that it's manageable, etc.

## Class Slack: 

HTTPS://USUTREETHINKING.SLACK.COM

Slack is a messenger/socialmedia/texting-like application that we’ll use for much of our outside-of-class communication. There's a channel there limited to students (no instructors!), channels for asking questions, etc.

## Course Texts

I have a class set of *Tree Thinking: An Introduction to Phylogenetic Biology* (Baum & Smith 2012); students are also encouraged to consult *Inferring Phylogenies* (Felsenstein 2004) and *Molecular Evolution: A Statistical Approach* (Yang 2014).


&nbsp;
## Schedule

### Jan 10 [All of us]

*Who are we, and want to do we want to get out of this seminar?*
* Introductions, goals (featuring our who-am-I slides)
* Discussion of format for course
* Texts, website, slack, sign-up sheet
* Outline of “phylogenetics”, i.e., introduction to trees—what is a phylogeny?—and tree thinking—what does this tree say?
* Introduction to Git
* Slides: [Introductions and why phylo?](https://github.com/crothfels/PhyloSeminar/blob/master/years/2024/presentations/PhyloSeminar_L1_whyPhylo.pptx)
* Slides: [introduction to git](https://github.com/crothfels/PhyloSeminar/blob/master/years/2024/presentations/git.pdf)


### Jan 17 [Carl]
*Three-hour tour, Part I: The Continuous Time Markov Model. The Core of the CTMM: transition rates, stationary frequencies, The Exponential Distribution*
* Reading (to be completed before class): chapters 1, 2, and 3 of _Tree Thinking_
* Slides: [PhyloIntro I](https://github.com/crothfels/PhyloSeminar/blob/master/years/2024/presentations/PhyloSeminar_L2.pptx). Remember: data have probabilities, models have likelihoods!


### Jan 24 [Carl]
*Three-hour tour, Part II: MCMCMCMC and tree models. In which we talk about Bayes Theorem, MCMC, and models that describe trees*
* Reading: chapters 8 and 9 of _Tree Thinking_
* Slides: [PhyloIntro II](https://github.com/crothfels/PhyloSeminar/blob/master/years/2024/presentations/PhyloSeminar_L3.pptx)


### Jan 31 [Carl]
*Three-hour tour, Part III: Potpourri. Do we believe our phylogenies, ILS, and divergence-time estimation*
* Reading: chapters 4, 6, 10, and 11 of _Tree Thinking_
* Slides: [PhyloIntro III](https://github.com/crothfels/PhyloSeminar/blob/master/years/2024/presentations/PhyloSeminar_L4.pptx)


### Feb 7 [David]
*Species delimitation*
* Reading: 
    * [How to Fail at Species Delimitation, Carstens et. al. 2013](https://github.com/crothfels/PhyloSeminar/blob/master/years/2024/readings/How_to_fail.pdf) 
    * [Species Delimitation and Relationships: The Dance of the Seven Veils, Naciri and Linder 2015](https://github.com/crothfels/PhyloSeminar/blob/master/years/2024/readings/Species_delim_and_rel.pdf)
* Slides: [Species Delimitation](https://github.com/crothfels/PhyloSeminar/blob/master/years/2024/presentations/Species_Delimitation.pptx)
* Bibliography: [Annotated Bibliography](https://github.com/crothfels/PhyloSeminar/blob/master/years/2024/annotatedBiblios/AnnotatedBibliography_Phyloseminar_David2024.pdf)

### Feb 14 [Gavin]
*Phylogenetic Analysis & Tree Building Software Sampler*
* Reading:
    * [An Introduction to Molecular Phylogenetic Analysis](https://github.com/crothfels/PhyloSeminar/blob/master/years/2024/readings/Horiike - 2016 - An Introduction to Molecular Phylogenetic Analysis.pdf)
    * [Phylogenetic tree building in the genomic age](https://github.com/crothfels/PhyloSeminar/blob/master/years/2024/readings/Kapli et al. - 2020 - Phylogenetic tree building in the genomic age.PDF)
* Slides: [Phylogenetic Tree Building](https://docs.google.com/presentation/d/14iqEO2M3pWpeqCBvX3HGA8VEtdeYGyzwKOwweRCf2fI/edit?usp=sharing)
    * [Demo Example Files](https://github.com/crothfels/PhyloSeminar/blob/master/years/2024/DemoExampleFiles)
* Bibliography: [Annotated Bibliography](https://github.com/crothfels/PhyloSeminar/blob/master/years/2024/annotatedBiblios/AnnotatedBibliography_Phyloseminar_GavinMunson2024.docx)

### Feb 21 [Julia H]
*Incongruence and ILS*
* Readings: 
    * [Extensive Genome-Wide Phylogenetic Discordance Is Due to Incomplete Lineage Sorting and Not Ongoing Introgression in a Rapidly Radiated Bryophyte Genus](https://github.com/crothfels/PhyloSeminar/blob/master/years/2024/readings/BryophyteILS.pdf)
    * [Disentangling Incomplete Lineage Sorting and Introgression to Refine Species-Tree Estimates for Lake Tanganyika Cichlid Fishes](https://github.com/crothfels/PhyloSeminar/blob/master/years/2024/readings/CyclidILS.pdf)
    * [Bonus paper - Incongruence in the Phylogenomic Era](https://github.com/crothfels/PhyloSeminar/blob/master/years/2024/readings/Incongruence.pdf)

* Slides:[Incomplete Lineage Sorting](https://github.com/crothfels/PhyloSeminar/blob/master/years/2024/presentations/ILS_Feb21.pptx)
* Bibliography: [ILS Annotated Bibliography](https://github.com/crothfels/PhyloSeminar/blob/master/years/2024/annotatedBiblios/PhyloSem_Incongruence_AnnBib_JuliaHobbie.docx)

### Feb 28 [Bhagya]
*Polymorphism-aware (PoMo) models*
* Reading:
   * [PoMo: AnAllele Frequency-Based Approach for Species Tree Estimation](https://github.com/crothfels/PhyloSeminar/blob/master/years/2024/readings/PoMo_Intro.pdf)
   * [Multi-level patterns of genetic structure and isolation by distance
 in the widespread plant Mimulus guttatus](https://github.com/crothfels/PhyloSeminar/blob/master/years/2024/readings/PoMo_Empirical.pdf)
* Slides: [PoMo](https://github.com/crothfels/PhyloSeminar/blob/master/years/2024/presentations/PoMo.pptx)
* Bibliography: [PoMo Annotated Bibliography](https://github.com/crothfels/PhyloSeminar/blob/master/years/2024/annotatedBiblios/PoMo_Annotatedv.docx)

### Mar 6 [Alivia]
*Identifying and Tracking TE with Bioinformatics/Phylogenetics*
* Reading: [A benchmark of transposon insertion detection tools using real data](https://github.com/crothfels/PhyloSeminar/blob/master/years/2024/readings/transposon_detection.pdf)
* Reading: [Using bioinformatic and phylogenetic approaches to classify transposable elements and understand their complex evolutionary histories](https://github.com/crothfels/PhyloSeminar/blob/master/years/2024/readings/evo_of_transp_elem.pdf)
* Slides:[Transposable Elements](https://github.com/crothfels/PhyloSeminar/blob/master/years/2024/presentations/Alivia Phylogenetics Seminar 2024.pptx)
* Bibliography: [Annotated Bibliography]()

### Mar 20 [Chinedum]
* Reading: []( )
* Slides: []()
* Bibliography: [Annotated Bibliography]()

### Mar 27 [Julia P.]
* Reading: []( )
* Slides: []()
* Bibliography: [Annotated Bibliography]()

### Apr 3 [Lauren H.]
*Morphometrics & Phylogenies*
* Reading: []( )
* Slides: []()
* Bibliography: [Annotated Bibliography]()

### Apr 10 [Onora]
* Reading: []( )
* Slides: []()
* Bibliography: [Annotated Bibliography]()

### Apr 17 [Aubry]
* Reading: []( )
* Slides: []()
* Bibliography: [Annotated Bibliography]()
